src/openforms/scss/components/admin/_logic-evaluated-table.scss

```diff
@@ -18,5 +18,15 @@
         width: 10%;
       }
     }
+    .logic-logs__row-body {
+      &--hidden {
+        display: none;
+      }
+    }
   }
 }
+
+#changelist-filter {
+  width: 10%;
+  margin: 0 !important;
+}
```

Comment:
it seems to me that this would also affect the styling on _different_ admin pages.

If you replicate the markup from the django changelist (including the filter classnames etc.), there should not be any need to override these styles.